content/influxdb/v2.0/reference/config-options.md

```diff
@@ -1469,13 +1470,64 @@ session-renew-disabled = true
 
 ---
 
+
+### sqlite-path
+
+Specify a user-defined path to the SQLite database file.
+The SQLite database is used to store metadata for the notebook and annotations features.
+
+| influxd flag    | Environment variable  | Configuration key |
+|:----------------|:----------------------|:------------------|
+| `--sqlite-path` | `INFLUXD_SQLITE_PATH` | `sqlite-path`     |
+
+**Default:** If not set explicitly, the default path for the SQLite database file
+is in the [`INFLUXD_BOLT-PATH`](#bolt-path) directory as `influxd.sqlite`.
```

Comment:
The default value should be before the table. Simplified the default description.

```suggestion
### sqlite-path

Path to the SQLite database file.
The SQLite database is used to store metadata for notebooks and annotations.

**Default:** _`influxd.sqlite` in the same directory as the [bolt-path](#bolt-path)._

| influxd flag    | Environment variable  | Configuration key |
|:----------------|:----------------------|:------------------|
| `--sqlite-path` | `INFLUXD_SQLITE_PATH` | `sqlite-path`     |
```